routes/router.js

```diff
@@ -29,4 +32,8 @@ module.exports = function(app, contextRoot) {
 
 	rootRouter.use('/api/team',teamAPI);
 
+	rootRouter.use('/api/project',projectAPI);
+
+    rootRouter.use('/api/us',userStoryAPI);
```

Comment:
us is not a good name, it is well known as a country name. better to use "story".